importer/importer.go

```diff
@@ -56,12 +60,28 @@ func Read_patient(path string) string {
 		patient.Diagnoses[i] = rawDiagnosesActive[i].(models.Diagnosis)
 	}
 
+	//diagnosis inactive
 	var diagnosisInactiveXPath = xpath.Compile("//cda:observation[cda:templateId/@root = '2.16.840.1.113883.10.20.24.3.13']")
 	rawDiagnosesInactive := ExtractSection(patientElement, diagnosisInactiveXPath, DiagnosisInactiveExtractor, "2.16.840.1.113883.3.560.1.2")
 	for i := range rawDiagnosesInactive {
 		patient.Diagnoses = append(patient.Diagnoses, rawDiagnosesInactive[i].(models.Diagnosis))
 	}
 
+	//lab results
+	var labResultXPath = xpath.Compile("//cda:observation[cda:templateId/@root = '2.16.840.1.113883.10.20.24.3.40']")
+	rawLabResults := ExtractSection(patientElement, labResultXPath, LabResultExtractor, "")
+	patient.LabResults = make([]models.LabResult, len(rawLabResults))
+	for i := range rawLabResults {
+		patient.LabResults[i] = rawLabResults[i].(models.LabResult)
+	}
+
+	//lab orders
+	var labOrderXPath = xpath.Compile("//cda:observation[cda:templateId/@root = '2.16.840.1.113883.10.20.24.3.37']")
+	rawLabOrders := ExtractSection(patientElement, labOrderXPath, LabOrderExtractor, "")
+	for i := range rawLabOrders {
+		patient.LabResults = append(patient.LabResults, rawLabOrders[i].(models.LabResult))
```

Comment:
Does storing `LabOrder`s as `LabResult`s allow us to store everything we extract from `LabOrder`? It looks like the `LabOrder` importer pulls out dates and such, but there's no date field in the `LabResult` model, at least that I can see.